src/theme.ts

```diff
@@ -237,8 +237,8 @@ export async function setTheme(theme?: string): Promise<void> {
 
     // look for the stylesheet elements.
     // styleElements is a map from style name to HTMLLinkElement.
-    const styleElements = Object.create(null);
-    const themes = Array.from(document.querySelectorAll('[data-mx-theme]'));
+    const styleElements: Record<string, HTMLLinkElement> = Object.create(null);
```

Comment:
Can you please switch this to a `Map` here?
Background is that we shouuld avoid `Record` types for key-based lookups + a `Map` may be faster.